src/coverage-system/sectionfinder.ts

```diff
@@ -1,5 +1,5 @@
 import {Section} from "lcov-parse";
-import {extname} from "path";
+import {basename} from "path";
```

Comment:
we were evening usign extname before?

Reply:
It wasn't being used according to VS Code and a quick search in the file.

Reply:
alright cool!